list.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'enumerable'
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] The _constructor_ of the `MyList` class is not able to accept multiple arguments. `MyList` class should be able to print output of a new initialization such as: `p MyList.new(4,6,8)`. Kindly ensure that the constructor of `MyList` class can arguments by implementing your constructor similar to this:
```ruby
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

Reply:
Thanks, @rloterh, for reviewing our pull request. Yes, it's a mistake. We thought that the input is an array. We will correct that and commit it again.